src/bin/pgcopydb/pgcmd.c

```diff
@@ -819,7 +819,16 @@ pg_restore_db(PostgresPaths *pgPaths,
 	args[argsIndex++] = (char *) pgPaths->pg_restore;
 	args[argsIndex++] = "--dbname";
 	args[argsIndex++] = (char *) connStrings->safeTargetPGURI.pguri;
-	args[argsIndex++] = "--single-transaction";
+
+	if(options.jobs == 1)
```

Comment:
Nitpicking: space between `if` and the open paren.

Reply:
Fixed.

I also went ahead and fix all the current indentation issues so that I can use citus_indent for formatting in this project. See https://github.com/dimitri/pgcopydb/pull/567